src/sentry/issues/escalating.py

```diff
@@ -343,12 +343,20 @@ def manage_issue_states(
             add_group_to_inbox(group, GroupInboxReason.ONGOING, snooze_details)
             record_group_history(group, GroupHistoryStatus.ONGOING)
     elif group_inbox_reason == GroupInboxReason.UNIGNORED:
-        updated = Group.objects.filter(
+        group = Group.objects.filter(
             id=group.id, status__in=[GroupStatus.RESOLVED, GroupStatus.IGNORED]
-        ).update(status=GroupStatus.UNRESOLVED, substatus=GroupSubStatus.ONGOING)
+        )
+        if not group.exists():
+            return
+
+        group = group.first()
```

Comment:
Nit: I think this will run two queries. You might be better off just using `.first()` and checking if `group is None`